e2e/test/scenarios/embedding/embed-resource-downloads.cy.spec.ts

```diff
@@ -120,6 +127,89 @@ describeWithSnowplowEE(
           export_type: "csv",
         });
       });
+
+      describe("with dashboard parameters", () => {
+        beforeEach(() => {
+          cy.signInAsAdmin();
+
+          setTokenFeatures("all");
+
+          // Test parameter with accentuation (metabase#49118)
+          const CATEGORY_FILTER = createMockParameter({
+            id: "5aefc725",
+            name: "usuário",
+            slug: "usu%C3%A1rio",
+            type: "string/=",
+          });
+          const questionDetails = {
+            name: "Products",
+            query: {
+              "source-table": PRODUCTS_ID,
+            },
+          };
+          createDashboardWithQuestions({
+            // Can't figure out the type if I extracted `dashboardDetails` to a variable.
```

Comment:
I don't know how to make extracting this to a variable works. I set it `as const` but TypeScript still complained, so I inline the value instead 🤷